es-ds-docs/pages/atoms/layout.vue

```diff
@@ -1,5 +1,328 @@
+                </b-col>
+                <b-col
+                    cols="2"
+                    class="bg-gray-dark">
+                    2 Columns
+                </b-col>
+                <b-col
+                    cols="2"
+                    class="bg-gray-light">
+                    2 Columns
+                </b-col>
+                <b-col
+                    cols="2"
+                    class="bg-gray-dark">
+                    2 Columns
+                </b-col>
+            </b-row>
+            <b-row>
+                <b-col
+                    cols="1"
+                    class="bg-gray-light">
+                    1
+                </b-col>
+                <b-col
+                    cols="1"
+                    class="bg-gray-dark">
+                    1
+                </b-col>
+                <b-col
+                    cols="1"
+                    class="bg-gray-light">
+                    1
+                </b-col>
+                <b-col
+                    cols="1"
+                    class="bg-gray-dark">
+                    1
+                </b-col>
+                <b-col
+                    cols="1"
+                    class="bg-gray-light">
+                    1
+                </b-col>
+                <b-col
+                    cols="1"
+                    class="bg-gray-dark">
+                    1
+                </b-col>
+                <b-col
+                    cols="1"
+                    class="bg-gray-light">
+                    1
+                </b-col>
+                <b-col
+                    cols="1"
+                    class="bg-gray-dark">
+                    1
+                </b-col>
+                <b-col
+                    cols="1"
+                    class="bg-gray-light">
+                    1
+                </b-col>
+                <b-col
+                    cols="1"
+                    class="bg-gray-dark">
+                    1
+                </b-col>
+                <b-col
+                    cols="1"
+                    class="bg-gray-light">
+                    1
+                </b-col>
+                <b-col
+                    cols="1"
+                    class="bg-gray-dark">
+                    1
+                </b-col>
+            </b-row>
+        </b-container>
+        <ds-doc-source
+            :doc-code="docCode"
+            doc-source="es-ds-docs/atoms/layout.vue" />
     </div>
 </template>
+<script setup lang="ts">
+import sassBreakpoints from '@energysage/es-bs-base/scss/modules/breakpoints.module.scss';
+import sassMaxWidths from '@energysage/es-bs-base/scss/modules/max-widths.module.scss';
+
+const  breakpointTableFields = [
+                { key: 'label', label: `` },
+                { key: 'xs', label: 'Extra small (xs)' },
+                { key: 'sm', label: 'Small (sm)' },
+                { key: 'md', label: 'Medium (md)' },
+                { key: 'lg', label: 'Large (lg)' },
```

Comment:
We don't have proper auto formatting set up yet but can these second-level lines be one tab in rather than four tabs in?